packages/contracts-bedrock/contracts/L1/SystemConfig.sol

```diff
@@ -117,7 +119,8 @@ contract SystemConfig is OwnableUpgradeable, Semver {
         uint64 _gasLimit,
         address _unsafeBlockSigner
     ) public initializer {
-        require(_gasLimit >= MINIMUM_GAS_LIMIT, "SystemConfig: gas limit too low");
+        uint256 maxResourceLimit = uint256(ResourceMetering(PORTAL).MAX_RESOURCE_LIMIT());
+        require(_gasLimit >= maxResourceLimit, "SystemConfig: gas limit too low");
```

Comment:
I'm a big fan of these changes. We may want to make sure to include the 1 Million gas used for the L1 Info Tx in this as well.

Reply:
Good catch